docs/ossm/ossm2-migration/network-policies.md

```diff
@@ -0,0 +1,144 @@
+
+## Network Policies created by 2.6:
+
+When `spec.security.manageNetworkPolicy=true`, the following Network Policies are created:
+
+### 1. Istiod Network Policy
+- **Purpose**: Controls incoming traffic to the webhook port of istiod pod(s)
+- **Location**: Created in SMCP namespace
+- **Sample YAML**:
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: istio-istiod-default    # Name format: istio-istiod-<revision>
+  namespace: istio-system       # Your SMCP namespace
+  labels:
+    maistra-version: "2.6.5"    # Version label
+    app: istiod                 # Identifies istiod component
+    istio: pilot                # Identifies as Istio pilot component
+    istio.io/rev: default       # Revision identifier
+    release: istio              
+  annotations:
+    "maistra.io/internal": "true"
+spec:
+  podSelector:
+    matchLabels:
+      app: istiod
+      istio.io/rev: default
+  ingress:
+    - ports:
+      port: webhook
+ ```
+
+### 2. Expose Route Network Policy
+- **Purpose**: Allows traffic from OpenShift ingress namespaces to pods labeled with `maistra.io/expose-route: "true"`
+- **Location**: Created in both:
+    - SMCP namespace
+    - Any namespace where ServiceMeshMember (SMM) is created
+- **Sample YAML**:
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: istio-expose-route-default   # Name format: istio-expose-route-<revision>
+  namespace: istio-system            # Your SMCP or member namespace
+  labels:
+    maistra-version: "2.6.5"
+    app: istio
+    release: istio
+spec: 
+  podSelector:
+    matchLabels:
+      maistra.io/expose-route: "true"
+  ingress:
+  - from:
+    - namespaceSelector:    # Allows traffic from OpenShift ingress
+        matchLabels:
+          network.openshift.io/policy-group: ingress
+```
+
+### 3. Default Mesh Network Policy
+- **Purpose**: Restricts traffic to pods only from namespaces explicitly labeled as part of the mesh (using label `maistra.io/member-of: <mesh-namespace>`)
+- **Location**: Created in both:
+    - SMCP namespace
+    - Any namespace where ServiceMeshMember is created
+- **Sample YAML**:
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: istio-mesh-default        # Name format: istio-mesh-<revision>
+  namespace: istio-system         # Your SMCP or member namespace
+  labels:
+    maistra-version: "2.6.5"
+    app: istio
+    release: istio
+spec:
+  ingress:
+  - from:
+    - namespaceSelector:                        # Only allows traffic from mesh members
+      matchLabels:
+        maistra.io/member-of: istio-system      # Replace with your SMCP namespace
```

Comment:
This one will need special description because the `maistra.io/member-of` label will be removed automatically after migration so if users are relying on that, it will break. We need to document this clearly here. It's mentioned partially in the cluster-wide doc [here](https://github.com/openshift-service-mesh/sail-operator/blob/main/docs/ossm/ossm2-migration/cluster-wide/README.md?plain=1#L29) so I think it should be explain here even with more details.

Reply:
https://github.com/openshift-service-mesh/sail-operator/pull/203/commits/19e47dd1d7ea6a8157f1a8d2b92a54909692a205

added additional note warning